package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "h5p.components-1.0",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Something short like "H5P Components library used for common content in different content types" or whatever ?

Reply:
I'll just use the same description as we're using in `library.json`:
> This component library can be used to quickly and easily get a unified look across different content types